config/default/migrate_plus.migration.hs_capx_images.yml

```diff
@@ -0,0 +1,88 @@
+uuid: c97e6ea0-2d0e-4eae-99b0-6d584c7ce942
+langcode: en
+status: true
+dependencies: {  }
+id: hs_capx_images
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags: {  }
+migration_group: hs_capx
+label: 'H&S CAPx Image Importer'
+source:
+  constants:
+    status: 1
+    type: image
+    file_destination: 'public://media/capx/'
+    file_extension: .jpg
+    separator: '-'
+  urls: {  }
+  item_selector: values
+  fields:
+    -
+      name: sunetid
+      label: sunetid
+      selector: uid
+    -
+      name: display_name
+      label: 'Display Name'
+      selector: displayName
+    -
+      name: alias
+      label: Alias
+      selector: alias
+    -
+      name: profile_photo
+      label: 'Profile Photo'
+      selector: profilePhotos/bigger/url
+    -
+      name: etag
+      label: 'CAP API Etag'
+      selector: meta/etag
+  ids:
+    sunetid:
+      type: string
+process:
+  name: display_name
+  status: constants/status
+  bundle: constants/type
+  image_timestamp:
+    -
+      plugin: skip_on_empty
+      method: row
+      source: profile_photo
+    -
+      plugin: explode
+      delimiter: '&'
+    -
+      plugin: str_replace
+      regex: true
+      search: '/^(?!ts=).+/'
+      replace: ''
+    -
+      plugin: str_replace
+      search: ts=
+      replace: ''
+    -
+      plugin: concat
+  image_path:
```

Comment:
As these are now entities in Drupal, are we possibly creating orphaned entities when the the image and timestamp changes and updates the parent node without revisions? Does the previous image get cleaned up or removed?

Reply:
If i'm not mistaken, the image on the media entity will get replaced. the old image will be cleaned up during garbage collection and the media entity will keep the same ID as the target from the person node.